### PR TITLE
Feat/email service

### PR DESCRIPTION
**Pull request by BartoszDolata**

### CODE REVIEW

**Comment by BartoszDolata** [comment]
Something went wrong with passing variables from .env.example. Tested on docker and email was sent :(

File: backend/src/main/java/com/example/cms/email/EmailSendingService.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+@Service
+public class EmailSendingService {
+    @Autowired
+    private JavaMailSender mailSender;
+
+    public void sendEmail(String receiver, String subject, String body){
+
+        SimpleMailMessage message = new SimpleMailMessage();
+        message.setFrom("test@mailpit");
```
Comment (AdrianKokot) [inline backend/src/main/java/com/example/cms/email/EmailSendingService.java:17]: This should use `SMTP_USERNAME` env variable

File: backend/pom.xml [hunk 2]
```diff
@@ -50,44 +50,53 @@
...
 		<plugins>
 			<plugin>
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
-    		</plugin>
+			</plugin>
 		</plugins>
-		
+
 		<resources>
```
Comment (AdrianKokot) [inline backend/pom.xml:91]: This section with resources can be deleted as angular application was moved to separate directory and is no longer inside spring project

File: backend/Dockerfile [hunk 3]
```diff
@@ -4,10 +4,18 @@ FROM maven:3.6.1-jdk-11-slim AS build
 ARG APPLICATION_SERVER
 ARG DATABASE_SCHEMA_HANDLING_ON_STARTUP
 ARG DATABASE_SCHEMA_CREATE_TYPE
+ARG SMTP_SERVER
```
Comment (AdrianKokot) [inline backend/Dockerfile:7]: This should not be needed, env variables are passed automatically to dockerfile during build, see [Dockerfile in root](https://github.com/AdrianKokot/put-reunice-platform/blob/main/Dockerfile), where I removed all `ARG ...` and `ENV ...` and environment variables are still recognized properly.

**Review by AdrianKokot — CHANGES_REQUESTED** [review]

**Comment by AdrianKokot** [comment]
> Something went wrong with passing variables from .env.example. Tested on docker and email was sent :(

Docker compose automatically searches for `.env` file, so either you should copy `.env.example` content to `.env` locally (it's marked in .gitignore to not push db passwords etc to remote) or add `--env-file` parameter when running docker, ex. `docker compose up -d --env-file .env.example`

**Review by AdrianKokot — APPROVED** [review]

**Pull request merged**